scripts/deploy.sh

```diff
@@ -103,10 +103,11 @@ restart_dj() {
     cd WcaOnRails
 
     # Kill all delayed_job workers (ignore if they were not running).
+    # This bit will disappear once we've switched to supervisor
```

Comment:
I've created https://github.com/thewca/worldcubeassociation.org/issues/983 for this and assigned it to you.